### PR TITLE
fix: [SIG-565]: design feedback for integrations

### PR DESCRIPTION
**Pull request by Vikrant2520**
### Summary

1. more spacing between remove integration and subtext
2. added correct icons for integrations tab
3. made the cards wider and move to next line if it overflows
4. Increase the list size to 20 
5. changed the URL with identifier to support back navigation 
6. added the dotted lines in test connection modal
7. implemented the ghost loading state

#### Related Issues / PR's

https://linear.app/signoz-io/issue/SIG-565/design-feedback-for-integrations-vqa

#### Screenshots

https://github.com/SigNoz/signoz/assets/54737045/f78f74b5-924a-4f15-b6cb-23df110b7dbc

### CODE REVIEW

File: frontend/public/Icons/group.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><g stroke="#C0C1C3" stroke-width="1.333" stroke-linecap="round" stroke-linejoin="round"><path d="M2 4.667V3.333C2 2.6 2.6 2 3.333 2h1.334M11.333 2h1.334C13.4 2 14 2.6 14 3.333v1.334M14 11.334v1.333C14 13.4 13.4 14 12.667 14h-1.334M4.667 14H3.333C2.6 14 2 13.4 2 12.667v-1.333M8.667 4.667H5.333a.667.667 0 00-.666.666v2c0 .368.298.667.666.667h3.334a.667.667 0 00.666-.667v-2a.667.667 0 00-.666-.667zM10.667 8H7.333a.667.667 0 00-.666.667v2c0 .368.298.666.666.666h3.334a.667.667 0 00.666-.666v-2A.667.667 0 0010.667 8z"/></g></svg>
```
Comment (YounixM) [inline frontend/public/Icons/group.svg:1]: Check in light mode

**Review by YounixM — APPROVED** [review]
Rest LGTM

File: frontend/public/Icons/group.svg (hunk 1 above)
Comment (Vikrant2520) [reply frontend/public/Icons/group.svg:1]: done ✅

**Comment by Vikrant2520** [comment]
image

image

**Review by YounixM — APPROVED** [review]

**Pull request merged**